src/modules/commentsModal.js

```diff
@@ -122,7 +128,41 @@ const generateModal = (id) => {
         li.appendChild(liText);
         ulComments.appendChild(li);
       });
-      // <li>Luis on Aug 30: Comment 3</li>
+    }
+  });
+
+  // Event listener for the submit button
+  submitBtn.addEventListener('click', () => {
+    if (nameInput.value === '' || commentTextArea.value === '') {
+      nameInput.classList.add('empty-input');
+      commentTextArea.classList.add('empty-input');
+      setTimeout(() => {
+        nameInput.classList.remove('empty-input');
+        commentTextArea.classList.remove('empty-input');
+      }, 1000);
```

Comment:
for example this statement you can finalise it just like this
``` 
if (nameInput.value === '' || commentTextArea.value === '') {
      nameInput.classList.add('empty-input');
      commentTextArea.classList.add('empty-input');
      setTimeout(() => {
        nameInput.classList.remove('empty-input');
        commentTextArea.classList.remove('empty-input');
      }, 1000);
     

            return;  // <-- this is the key
```

so as soon as your code has something empty you, stop the execution of the function up to that point and in that way you can avoid the `else` statement that enclose everything that will happen if you don't have empty values

Reply:
Thank you for the heads up! I'll take a look into that.